README.md

```diff
@@ -6,7 +6,7 @@
 
 Julia package for causal inference, graphical models and structure learning.
 
-This package contains code for the stable version of the PC algorithm and the extended FCI algorithm.
+This package contains code for the stable version of the PC algorithm and the extended FCI algorithm, the score based GES algorithm and intervention search.
```

Comment:
You mean adjustment set search? Or what do you mean with "intervention search"?

Reply:
Yes, thanks